expected/13.12/extra/aggregates.out

```diff
@@ -467,43 +532,124 @@ DELETE FROM agg_t3;
 --Testcase 332:
 INSERT INTO agg_t3 VALUES ('1'::float8), ('infinity'::float8);
 --Testcase 333:
-SELECT avg(a), var_pop(a) FROM agg_t3;
-   avg    | var_pop 
-----------+---------
- Infinity |     NaN
+SELECT sum(a), avg(a), var_pop(a) FROM agg_t3;
+   sum    |   avg    | var_pop 
+----------+----------+---------
+ Infinity | Infinity |     NaN
 (1 row)
 
 --Testcase 334:
 DELETE FROM agg_t3;
 --Testcase 335:
 INSERT INTO agg_t3 VALUES ('infinity'::float8), ('1'::float8);
 --Testcase 336:
-SELECT avg(a), var_pop(a) FROM agg_t3;
-   avg    | var_pop 
-----------+---------
- Infinity |     NaN
+SELECT sum(a), avg(a), var_pop(a) FROM agg_t3;
+   sum    |   avg    | var_pop 
+----------+----------+---------
+ Infinity | Infinity |     NaN
 (1 row)
 
 --Testcase 337:
 DELETE FROM agg_t3;
 --Testcase 338:
 INSERT INTO agg_t3 VALUES ('infinity'::float8), ('infinity'::float8);
 --Testcase 339:
-SELECT avg(a), var_pop(a) FROM agg_t3;
-   avg    | var_pop 
-----------+---------
- Infinity |     NaN
+SELECT sum(a), avg(a), var_pop(a) FROM agg_t3;
+   sum    |   avg    | var_pop 
+----------+----------+---------
+ Infinity | Infinity |     NaN
 (1 row)
 
 --Testcase 340:
 DELETE FROM agg_t3;
 --Testcase 341:
 INSERT INTO agg_t3 VALUES ('-infinity'::float8), ('infinity'::float8);
 --Testcase 342:
-SELECT avg(a), var_pop(a) FROM agg_t3;
- avg | var_pop 
------+---------
- NaN |     NaN
+SELECT sum(a), avg(a), var_pop(a) FROM agg_t3;
+ sum | avg | var_pop 
+-----+-----+---------
+ NaN | NaN |     NaN
+(1 row)
+
+--Testcase 603:
+DELETE FROM agg_t3;
+--Testcase 604:
+INSERT INTO agg_t3 VALUES ('-infinity'::float8), ('-infinity'::float8);
+--Testcase 605:
+SELECT sum(a), avg(a), var_pop(a) FROM agg_t3;
+    sum    |    avg    | var_pop 
+-----------+-----------+---------
+ -Infinity | -Infinity |     NaN
+(1 row)
+
+--Testcase 606:
+DELETE FROM agg_t3;
+--Testcase 607:
+INSERT INTO agg_t3 VALUES ('1'::numeric), ('infinity'::numeric);
+ERROR:  invalid input syntax for type numeric: "infinity"
+LINE 1: INSERT INTO agg_t3 VALUES ('1'::numeric), ('infinity'::numer...
```

Comment:
Is this expectation correct?

Reply:
I think yes. Because https://www.postgresql.org/docs/14/datatype-numeric.html and https://www.postgresql.org/docs/13/datatype-numeric.html shows a difference.
14

> In addition to ordinary numeric values, the **numeric** type has several special values:

13

> In addition to ordinary numeric values, the **floating-point** types have several special values:

Reply:
Thank you for supporting. It's OK